README.md

```diff
@@ -18,6 +18,18 @@ Export your GitHub API token:
 
     export GITHUB_API_TOKEN="your-organization-github-token-here"
 
+### Github Omniauth Integration:
+     
+    You must create a developer application to use omniauth.
+    If you do NOT add the appropriate env keys below AUTH will be OFF I.E => Fail unsafe. Please be carefull and check the logs. 
+    Please  note the "Authorization callback URL" should match your site i.e. "http://<your-url-and-port>/auth/github/callback"
+
+    export GITHUB_RANDOM_HASH_KEY="some-random-hash-here;  
+    export GITHUB_OMNI_AUTH_ID="github-application-is";
+    export GITHUB_OMNI_AUTH_KEY="github-application-key";
+    export GITHUB_CALLBACK_URL="http://<your-url-and-port";  // must match that given to github! Would be 127.0.0.1:port for testing
```

Comment:
Inferred in the code as I needed to have different full url paths. 
i.e.
http.Redirect(w, r, os.Getenv("GITHUB_CALLBACK_URL")+"/auth/github/login"